### ISSUE TITLE
Better fonts for Syriac images

### ISSUE THREAD

**Issue by n8willis** [n8willis/opentype-shaping-documents#145]
Noto Sans for Syriac is so monoline and geometric that I worry that the images aren't helpful.

It'd be good to find a better replacement and regenerate them. Suggestions are welcome, given that we intentionally want to use compatibly-licensed fonts so that others can rebuild everything from source.

**Comment by alerque** [comment]
I don't know if it's a starting point for someone with more knowledge of Syriac than I, but I had this page bookmarked as a starting point for when I needed fonts for some related languages:

http://luc.devroye.org/assyrian.html

**Comment by alfiedotwtf** [comment]
Here are the fonts we use for `Allsorts` Syriac testing. Hopefully one of these will be what you're after:

- East Syriac Adiabene
- East Syriac Ctesiphon
- Estrangelo Antioch
- Estrangelo Edessa
- Estrangelo Midyat
- Estrangelo Nisibin
- Estrangelo Nisibin Outline
- Estrangelo Quenneshrin
- Estrangelo Talada
- Estrangelo TurAbdin
- Serto Batnan
- Serto Jerusalem
- Serto Jerusalem Outline
- Serto Kharput
- Serto Malankara
- Serto Mardin
- Serto Urhoy

**Comment by n8willis** [comment]
> Here are the fonts we use for `Allsorts` Syriac testing. Hopefully one of these will be what you're after:

Yeah, the Beth Mardutho fonts are what a lot of people seem to use for reference quality (and the folks there were also quite helpful in answering questions). The singleton license-statement/terms in the associated documentation and embedded with some of the fonts themselves is where I haven't figured out exactly how they shake out. There's some mutual contradictions on things like whether redistribution is permitted.
